src/protocol/protocol.yml

```diff
@@ -380,18 +380,42 @@ Playwright:
 
   commands:
 
-    setForwardedPorts:
+    setSocketLocalAddress:
```

Comment:
this now pollutes the global Playwright object which I was trying to avoid.

Reply:
I prefixed those with socks. That's what we do for page.mouse/keyboard/tap, etc.